.github/workflows/zcash-params.yml

```diff
@@ -3,8 +3,8 @@ name: zcash-params
 on:
   workflow_dispatch:
   push:
-    branches:
-      - 'main'
+    # branches:
+    #   - 'main'
```

Comment:
Question: What does this change do? Should we just delete these lines instead?

Reply:
This is just temporal so the job runs each time I push, to validate it's building correctly. I can't merge this. Next time I'll add a TODO so it's clearer.